src/api/mod.rs

```diff
@@ -17,8 +17,8 @@ use crate::{common::Cluster, util::rpc_url_from_cluster};
 use solana_client::rpc_client::RpcClient;
 use crate::request_handler::RequestHandler;
 
-const API_URL_V1: &str = "https://api.helius.xyz/v1";
-const API_URL_V0: &str = "https://api.helius.xyz/v0";
+const API_URL_V1: &str = "https://api-mainnet.helius-rpc.com/v1";
+const API_URL_V0: &str = "https://api-mainnet.helius-rpc.com/v0";
```

Comment:
Taken from typescript sdk 
https://github.com/helius-labs/helius-sdk/blob/main/src/utils/get-helius-endpoints.ts

Reply:
Oh interesting I didn't noticed they changed those, thanks!

Could you please update the devnet api url, as well as the rpc endpoints defined in `rpc_url_from_cluster`?

Reply:
Done

> Oh interesting I didn't noticed they changed those, thanks!
> 
> Could you please update the devnet api url, as well as the rpc endpoints defined in `rpc_url_from_cluster`?

done

Reply:
Thank you kindly